lib/isolator/adapters/action_mailer.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+ActiveSupport.on_load(:action_mailer) do
+  Isolator.isolate :action_mailer, ActionMailer::Base.singleton_class,
```

Comment:
What if make this adapter _Rails-less_ and directly track `mail` gem (`Mail::Message#deliver` method)?

That should work with other frameworks too (like Hanami), 'cause everyone uses `mail` under the hood.

No need to rewrite specs, btw; maybe, add a _raw_ `Mail::Message` example.

Reply:
@palkan  Good point, I'll try!

Reply:
@palkan Check it out https://github.com/palkan/isolator/pull/5/commits/92f9329758a1fcf2bf550a8f6b56f1fe99a5f9e1